web/src/test/auto/headless/engine/keyboard/transcriptions.js

```diff
@@ -1,6 +1,6 @@
 import { assert } from 'chai';
 
-import { Mock, findCommonSubstringEndIndex } from '@keymanapp/keyboard-processor';
+import { Mock, findCommonSubstringEndIndex } from 'keyman/engine/keyboard';
```

Comment:
This test focuses on Mocks and the Transcription subsystem, both of which should be landing within `engine/js-processor`.